code/game/objects/structures/aliens.dm

```diff
@@ -286,7 +282,9 @@
         check_counter = 0
 
 /obj/structure/alien/weeds/proc/clear_wall_weed()
-	QDEL_NULL(wall_weed)
+	if(wall_weed)
+		wall_weed.silent_removal = TRUE
+		QDEL_NULL(wall_weed)
```

Comment:
```suggestion
	if(wall_weed && !QDELETED(wall_weed))
		wall_weed.silent_removal = TRUE
	QDEL_NULL(wall_weed)
```
Just in case something else deleted them first.